src/prelude.rs

```diff
@@ -1,4 +1,4 @@
-//! A convenience module that re-exports the most commonly-used Neon APIs.
+//! Convenience module for the most common Neon imports.
```

Comment:
Should we consider removing this or refactoring as part of `1.0`? It includes nearly everything in `neon`.

Reply:
I kind of like the prelude idiom but I'm happy to discuss it! Let's discuss on Slack or in a separate issue thread.